modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,36 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$disabled_functions = explode( ',', ini_get( 'disable_functions' ) );
+		
+		// Avoid warnings when print_r() has been disabled on hosting provider for security reasons
+		if ( ! in_array( 'print_r', $disabled_functions ) ) {
+			$all_values_string = print_r( $all_values, true );
+		}
+		// If print_r is disabled, try to use var_export as an alternative solution
+		elseif ( ! in_array( 'var_export', $disabled_functions ) ) {
```

Comment:
As per [WP Coding Style](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style), conditionals should be on the same line as the prior closing brace.  So this line should be:

```php
} elseif ( ! in_array( 'var_export', $disabled_functions ) ) {
```

with the comment either at the end of the line, or below and indented.